src/icons/f5.svg

```diff
@@ -0,0 +1,16 @@
+<svg version="1.2" baseProfile="tiny-ps" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 640" width="640" height="640">
```

Comment:
Can be reduced by at least 25%
https://jakearchibald.github.io/svgomg/

Reply:
I think since its vector, the size doesn't matter too much. But I will reduce it tomorrow

Reply:
I'm not sure I follow the logic regarding the size of a file not mattering because it contains vector data, but as long as you are willing to optimize it that's fine :)

Reply:
SVGs are just points with fill info, so image size, 640x640 vs 128x128, is the same amount of data, and file size wont change.
however, that tool you linked found ways to reduce the size! nifty! so i not only reduced the image size but also ran the compression.